modules/hcp-create-infraenv.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-infraenv_{context}"]
+= Create Infraenv
+
+For heterogeneous node pools, you must create the `infraenv` custom resource (CR) for each architecture. For example, for node pools with `x86_64` and `ppc64le` architectures, create the `infraenv` CR for `x86_64` and `ppc64le`.
+
+.Procedure
+
+. Create the `infraenv` resource for heterogeneous node pools by running the following command:
```

Comment:
This is two commands. Each one should have a step, and the step needs to explain what each does.